docs/technical/01_addresses.md

```diff
@@ -142,10 +142,14 @@
 
 ## Multisigs
 
-| Contract | Address                                                                                                                 |
-| -------- | ----------------------------------------------------------------------------------------------------------------------- |
-| DAO      | [`0x245cc372C84B3645Bf0Ffe6538620B04a217988B`](https://etherscan.io/address/0x245cc372C84B3645Bf0Ffe6538620B04a217988B) |
-| Policy   | [`0x0cf30dc0d48604A301dF8010cdc028C055336b2E`](https://etherscan.io/address/0x0cf30dc0d48604A301dF8010cdc028C055336b2E) |
+| Contract | Chain     | Address                                                                                                                            |
+| -------- | --------- | ---------------------------------------------------------------------------------------------------------------------------------- |
+| DAO      | Ethereum  |[`0x245cc372C84B3645Bf0Ffe6538620B04a217988B`](https://etherscan.io/address/0x245cc372C84B3645Bf0Ffe6538620B04a217988B)             |
+| DAO      | Arbitrum  |[`0x012BBf0481b97170577745D2167ee14f63E2aD4C`](https://arbiscan.io/address/0x012BBf0481b97170577745D2167ee14f63E2aD4CB)             |
```

Comment:
url should be https://arbiscan.io/address/0x012BBf0481b97170577745D2167ee14f63E2aD4C
```suggestion
| DAO      | Arbitrum  |[`0x012BBf0481b97170577745D2167ee14f63E2aD4C`](https://arbiscan.io/address/0x012BBf0481b97170577745D2167ee14f63E2aD4C)             |
```